src/themes/demo/fill-canvas.twig

```diff
@@ -0,0 +1,16 @@
+<div{% if theme %} class="t-{{theme}}"{% endif %} style="
```

Comment:
Might be nice to incorporate some of the PR context into a comment here for future devs

Reply:
I've added a comment!